cluster/gce/config-common.sh

```diff
@@ -85,15 +85,15 @@ function get-node-ip-range {
 }
 
 function get-cluster-ip-range {
-  local suggested_range="10.100.0.0/14"
+  local suggested_range="10.160.0.0/14"
```

Comment:
This seems to intersect with the range reserved for auto subnets (10.128.0.0/9). Could we choose some other ranges instead?

Ref: https://github.com/kubernetes/kubernetes/pull/44509

cc @bowei

Reply:
10.32.0.0 seems like a good choice - i.e it doesn't intersect with the auto subnets and is also valid for /11 mask (which is what we use for > 4k node clusters)

Reply:
Discussed offline with @bowei. This should be fine.

Reply:
10.32.0.0/11 intersects with the node ip range though...

https://github.com/kubernetes/kubernetes/blob/bacc01f7295d0df75cdbb52e7e63e7fe3fb883f4/cluster/gce/config-common.sh#L68-L85

Reply:
That's sad. 10.64.0.0 should be fine right? Doesn't intersect with 10.40.0.0/19 and 10.128.0.0/9 and is also valid for /11 mask. SG?